src/palakSJun20/ReverseIntegerNumbers.java

```diff
@@ -0,0 +1,25 @@
+/*Program 3: WAP to reverse number (Consider Input as Integer not as String).
+Example:-
+Input:- 12345
+Output:- 54321*/
+package palakSJun20;
+
+public class ReverseIntegerNumbers {
+
+	//Method to Reverse the integer Numbers
+	void reverseInteger(int number) {
+		int reverse = 0;
+		int temp = number;
+		while (temp > 0) {
+			reverse = temp % 10;
+			temp = temp / 10;
+			System.out.print(reverse);
+		}
```

Comment:
The output is exactly as expected.
But, here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your login.

Reply:
Changes Updated.